docs/proposals/20200511-clusterctl-extensible-template-processing.md

```diff
@@ -62,12 +62,12 @@ Some templating tools that can be used to manage your templates.
 * Cue - Cue Data Constraint Language.
   * Source: https://github.com/cuelang/cue
 * Dhall - Dhall Programming Configuration Language.
-  * Source: https://github.com/dhall-lang/dhall-lang/blob/master/README.md
+  * Source: https://github.com/dhall-lang/dhall-lang
```

Comment:
This change technically changes the link, we are now pointing to the repo root rather than the readme explicitly. Does this matter?

Reply:
I don't think it really matters as the repo root also shows the README, but it's totally fine for me if we want to change it back

Reply:
I don't really have a strong opinion, but my thought was basically, when this was introduced, was there a deliberate intention to make it point to the readme file directly 🤔

Reply:
Yup, let's see if anyone else remembers :)